src/app/components/Embeds/EmbedError/index.styles.tsx

```diff
@@ -19,11 +19,19 @@ const styles = {
       ...fontVariants.sansBold,
       paddingTop: `${spacings.FULL}rem`,
     }),
-  errorMessage: ({ fontVariants }: Theme) =>
+  errorMessage: ({ fontVariants, palette, fontSizes }: Theme) =>
     css({
+      color: palette.GREY_10,
       ...fontVariants.sansRegular,
+      ...fontSizes.longPrimer,
```

Comment:
May be something we can add later, but we have a `<Text />` component that is polymorphic, so you can set it to any valid HTML element as well as specifying the font variant and size, e.g:
```
    <Text as="strong" fontVariant="sansRegular" size="longPrimer">
      {message}
    </Text>
```

Just saves having to set the font styling in CSS.

Reply:
Oh yh good spot!